runner/src/load_generator/utils/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod ceramic_models_utils;
```

Comment:
it might be nicer to export the types you want and just refer to `load_generator::utils::CeramicConfig` etc and remove the extra module in the path

rather than having a file called `generator_utils.rs` in a module called generator_utils, I'd probably end up putting it in mod.rs. that said, I generally prefer the lib/mod file to just be exports with all the code living in its own file, so I'd suggest at least erasing the name by exporting types directly. basically, you could change

```rust
pub mod ceramic_models_utils;
pub mod generator_utils;
```
to 
```rust
mod ceramic_models_utils;
mod generator_utils;

// pub use ::* isn't the best practice, but makes sense in a utils crate
pub use ceramic_models_utils::*;
pub use generator_utils::*;
```
and then in other places you'd just refer to `load_generator::utils::X`.. if models/generator stuff are distinct enough, they might merit their own modules rather than being deep inside a utils module